packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -296,10 +297,10 @@ module.exports = {
           // that fall through the other loaders.
           {
             // Exclude `js` files to keep "css" loader working as it injects
-            // it's runtime that would otherwise processed through "file" loader.
+            // its runtime that would otherwise processed through "file" loader.
```

Comment:
Wayyyy above here on L98-99, there's a require of `./polyfills` that looks like it's only in our fork.  Is this something that we added, or is it something they removed a while back that we missed?  If the former, we should add a `ZEAL` comment; if the latter, we should probably also remove it.

Reply:
It's still there, just out of view for the comparison. It's their's

https://github.com/facebook/create-react-app/compare/v1.1.4...CodingZeal:chores/cra-1.1.4#diff-a7f98c18479be87c9f33e7604dbd1a09L45

Reply:
So it is.  Should we move ours up to the same place as theirs to avoid this showing as a weird diff?

Reply:
Actually it is duplicated now that I look closer so I have removed it.